jpa/src/main/java/org/jdbi/v3/jpa/AnnoMapper.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.jpa;
+
+import org.jdbi.v3.StatementContext;
+import org.jdbi.v3.tweak.ResultColumnMapper;
+import org.jdbi.v3.tweak.ResultSetMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.persistence.Entity;
+import java.lang.reflect.Constructor;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class AnnoMapper<C> implements ResultSetMapper<C> {
+
+    private final Class<C> clazz;
+    private final AnnoClass<C> annos;
+
+    public static boolean accept(Class<?> clazz) {
+        if (logger.isDebugEnabled()) {
+            logger.debug("accept " + clazz);
```

Comment:
not sure what this debug is here for, if we want more info about the mapping process maybe we should improve the logging in core instead